core/src/index.ts

```diff
@@ -11,6 +11,7 @@ export { componentOnReady } from './utils/helpers';
 export { isPlatform, Platforms, PlatformConfig, getPlatforms } from './utils/platform';
 export { IonicSafeString } from './utils/sanitization';
 export { IonicConfig, getMode, setupConfig } from './utils/config';
+export { openURL } from "./utils/theme";
```

Comment:
The code formatting here will fail our prettier diff check due to the use of double quotes instead of single quotes.

In the `packages/core` project, you can run `npm run prettier -- --write` and commit the diff.

Reply:
Thanks! I've pushed the changes now